client/integrations/integrations.go

```diff
@@ -804,6 +810,120 @@ func getVersionId(name string, filter string) (version string, err error) {
 	}
 }
 
+// ExportWithThreads exports all Integration Flows in the specified folder using a configurable number of threads
+func ExportWithThreads(folder string, numThreads int) error {
+	// Set export settings
+	apiclient.SetExportToFile(folder)
+	apiclient.SetPrintOutput(false)
```

Comment:
i think you should copy this to a `printSetting`. 

```
printSetting := apiclient.GetPrintOutput()
apiclient.SetPrintOutput(false)
```

and in the end, restore `printSetting`

Reply:
due to goroutines this is not working as expected.